### PR TITLE
Use Auto Assign Action to assign one TC member

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

**Comment by bogdandrutu** [comment]
Not yet working, will give more tries

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Set to true to add assignees from different groups to pull requests
+useAssigneeGroups: true
+
+# A list of assignees, split into different groups, to be added to pull requests (GitHub user name)
+assigneeGroups:
+    tc:
+        - bogdandrutu
+        - carlosalberto
+        - SergeyKanzhelev
+        - yurishkuro
```
Comment (tigrannajaryan) [inline .github/auto_assign.yml:16]: How does this work? Are assignments done in round-robin manner?

**Comment by SergeyKanzhelev** [comment]
@bogdandrutu is it tested and ready to merge?

**Comment by SergeyKanzhelev** [comment]
Let's try it. Validated again config looks OK. Merging

**Pull request merged**